client/web/src/components/diff/FileDiffNode.tsx

```diff
@@ -97,10 +97,21 @@ export const FileDiffNode: React.FunctionComponent<React.PropsWithChildren<FileD
     return (
         <>
             {/* The empty <a> tag is to allow users to anchor links to the top of this file diff node */}
-            <Link to="" id={anchor} aria-hidden={true} />
-            <div className={classNames('test-file-diff-node', styles.fileDiffNode, className)}>
+            {/*
+                a11y-ignore
+                Rule: "aria-hidden-focus" (ARIA hidden element must not contain focusable elements)
+                GitHub issue: https://github.com/sourcegraph/sourcegraph/issues/33343
```

Comment:
```suggestion
                GitHub issue: https://github.com/sourcegraph/sourcegraph/issues/33343
```

This issue isn't relevant here